tests/test_core.py

```diff
@@ -4579,9 +4581,7 @@ def test_dylink_argv_argc(self):
   def test_dylink_weak(self):
     # Verify that weakly defined symbols can be defined in both side module and main
     # module but that only one gets used at runtime.
-    main = test_file('core/test_dylink_weak_main.c')
-    side = test_file('core/test_dylink_weak_side.c')
-    self.dylink_testf(main, side, force_c=True, need_reverse=False)
```

Comment:
`force_c=True` was deleted in the RHS.. Is this intended?

Reply:
Yes, force_c is only needed when the filename is not specified.  In this case it doesn't do anything.